js/main.js

```diff
@@ -63,6 +63,12 @@ function renderImages() {
     commentInput.setAttribute('type', 'text');
     commentInput.setAttribute('placeholder', 'Add comments here');
     commentInput.setAttribute('id', 'comment-box');
+    if (imageValues[i].editing === true) {
+      commentOutput.className = 'hidden';
+    }
+    if (commentObj) {
+      commentInput.value = commentOutput.textContent;
+    }
```

Comment:
Did a lot of code for this feature accidentally get merged at some point? I can't imagine this was the only additional JavaScript necessary to implement editing